src/lib/cache/subCache.cpp

```diff
@@ -1285,3 +1306,33 @@ void subCacheStart(void)
   }
   pthread_detach(tid);
 }
+
+
+
+/* ****************************************************************************
+*
+* subCacheItemErrorStatus - 
```

Comment:
errors in the cache itself .... note the word `Item` as part of the function name.
Function explained in function header and name changed to subCacheItemNotificationErrorStatus in 8eaaca3